src/components/Input.jsx

```diff
@@ -0,0 +1,11 @@
+const Input = ({ placeholder, className, value, onChange }) => {
```

Comment:
💊 제안
className을 받으시고 빈 문자열로 초기화해주셔야 className에 undefined가 들어가지 않습니다~

```suggestion
const Input = ({ placeholder, className = "", value, onChange }) => {
```